toolkit/scss/_proposition-header.scss

```diff
@@ -17,3 +17,69 @@
 .logout-button:hover {
     text-decoration: underline;
 }
+
+#global-header.with-proposition {
+    .header-wrapper {
+        .header-global {
+            width: 50%;
+
+            @media only screen and (max-width: 767px)  {
+                width: 100%;
+            }
+
+            .header-logo {
+                width: auto;
+            }
+
+            .header-title {
+                float: left;
+                font-family: "nta", Arial, sans-serif;
```

Comment:
Can we be making use of our variables here? Above is using `$toolkit-font-stack;`